manage/installing/installation_windows10_wsl.rst

```diff
@@ -0,0 +1,271 @@
+Install Pyenv
+-------------
+
+To install Pyenv, use the command below in the Ubuntu shell.
+After completing the download, the shell has to be restarted to take over all changes.
+
+.. code-block :: console
+
+    curl https://pyenv.run | bash
+
+Configure Paths
+---------------
+
+To set the right paths for certain activities and enable Pyenv local environment, edit the file ".bashrc" in Ubuntu using the "nano" editor command.
+
+.. code-block :: console
+
+    nano .bashrc
+
+After opening the document, at the end (bottom of code) 3 lines have to be appended:
+
+.. code-block :: bash
+
+    export PATH="$HOME/.pyenv/bin:$HOME/.local/bin:$PATH"
+    eval "$(pyenv init -)"
+    eval "$(pyenv virtualenv-init -)"
+
+The path is important insofar, because further commands can executed if the paths, where they have to be tapped, are known.
+To apply the changes, save the document and activate them by typing the following into the Ubuntu shell.
+
+.. code-block :: console
+
+    source .bashrc
+
+Install Python
+--------------
+
+After finishing the setup, it is possible to install the needed version of Python in Ubuntu.
+Before installing Python, the system need to be prepared with a C-compiler and some libraries.
+To install them, a sudo (superuser do) command (with password, configured when setting up Ubuntu shell) is needed.
+
+.. code-block :: console
+
+    sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+In this case, Python version 3.7.4 is installed.
+To use the language version globally, it has to be declared first.
+
+.. code-block :: console
+
+    pyenv install 3.7.4
+    pyenv global 3.7.4
+
+Plone CLI
+=========
+
+We use Plone CLI here to work with Plone.
+Plone CLI is a command line interface for creating Plone packages, as also for building and starting Plone.
+Plone CLI need to be installed on the subsystem by typing the following command.
+
+It is installed as a global user-package, so that it can be used for several projects.
+Plone CLI's newest release will be pulled immediately.
+While at it we install also the latest Pip first.
+Pip is a Python package installer.
+It pulls released Python packages from the `Python Package Index <https://pypi.org/>`_ and installs them in the current Python environment.
+
+.. code-block :: console
+
+    pip install --upgrade pip
+    pip install plonecli --user
+
+Bash Auto Completion
+--------------------
+
+To activate the autocomplete function for Plone CLI, again the ``.bashrc`` document has to be opened and a path is inserted ate the bottom of the so far code.
+
+Open editor.
+
+.. code-block :: console
+
+    nano .bashrc
+
+Code to insert.
+
+.. code-block :: bash
+
+    . ~/.local/bin/plonecli_autocomplete.sh
+
+Afterwards, the script has to be applied again.
+
+.. code-block :: console
+
+    source .bashrc
```

Comment:
I don't think this is a good idea to force the execution of a script upon opening a new shell every time. What if the user only wants to login and and do something else?

Can you provide an explanation as to why you chose to put it in `.bashrc` instead of manually executing the script?